src/modules/microdds_client/dds_topics.yaml

```diff
@@ -84,3 +208,25 @@ subscriptions:
 
   - topic: /fmu/in/vehicle_trajectory_waypoint
     type: px4_msgs::msg::VehicleTrajectoryWaypoint
+
+  - topic: /fmu/in/actuator_outputs
+    type: px4_msgs::msg::ActuatorOutputs
+
+  - topic: /fmu/in/actuator_servos
+    type: px4_msgs::msg::ActuatorServos
+   
+  - topic: /fmu/in/actuator_servos_trim
+    type: px4_msgs::msg::ActuatorServosTrim
+    
+  - topic: /fmu/in/vehicle_global_position
+    type: px4_msgs::msg::VehicleGlobalPosition
+  
+  - topic: /fmu/in/vehicle_local_position
+    type: px4_msgs::msg::VehicleLocalPosition
+    
+  - topic: /fmu/in/led_control
+    type: px4_msgs::msg::LedControl
```

Comment:
Is this really necessary to have?

Reply:
Hello, for me and perhaps for other people yes... If i want to make a kalman in my raspbery pi with ros2 to fusion with others imu for exemple.

Reply:
@ozitoune I can see why you want it, but making this as a default would be a waste of bandwidth

Reply:
ah ok, topics take bandwidth if also is they no subscribed ?